cli/kata-env_amd64_test.go

```diff
@@ -14,7 +14,10 @@ import (
 )
 
 func getExpectedHostDetails(tmpdir string) (HostInfo, error) {
-	return genericGetExpectedHostDetails(tmpdir)
+	expectedVendor := "moi"
+	expectedModel := "awesome XI"
+	return genericGetExpectedHostDetails(tmpdir, expectedVendor, expectedModel)
+	//	return genericGetExpectedHostDetails(tmpdir)
```

Comment:
Please remove all the commented out code on this PR.

Reply:
@jodh-intel : Oops, sorry for that. I have re-pushed the PR.